lib/Graph/Graph.cpp

```diff
@@ -38,14 +38,18 @@ using llvm::isa;
 
 namespace {
 /// A helper function to log the deletion of constant/placeholder \p s of a
-/// module.
-void loggingStorageDeletionInUserFunc(Storage *s) {
-  std::unordered_set<Function *> userFuncs;
-  for (auto &use : s->getUsers()) {
-    userFuncs.insert(use.getUser()->getParent());
+/// module into the log context of given functions \p functions.
```

Comment:
Can you add a note explaining why we can't do it the way we were before, i.e. because all users have already been removed hence why we can remove it in the first place.